docs/ota-client-guide/modules/ROOT/pages/add-ota-functonality-existing-yocto-project.adoc

```diff
@@ -13,11 +13,16 @@ endif::[]
 :page-order: 6
 :icons: font
 
-If you already have a Yocto-based project that you want to update using {product-name}, you just need to do four things to get started:
+If you already have a Yocto-based project that you want to update using {product-name}, you can perform the four things outlined below. These are the main changes needed to start a functional integration to an existing Yocto project.
 
 1.  Clone the https://github.com/advancedtelematic/meta-updater[meta-updater] layer and add it to your https://www.yoctoproject.org/docs/2.6/ref-manual/ref-manual.html#structure-build-conf-bblayers.conf[bblayers.conf].
 2.  Clone a BSP integration layer (`meta-updater-$\{PLATFORM}`, e.g. https://github.com/advancedtelematic/meta-updater-raspberrypi[meta-updater-raspberrypi]) and add it to your conf/bblayers.conf. If your board isn't supported yet, you could write a BSP integration for it yourself. See the <<Adding support for your board>> section for the details.
-3.  Set up your https://www.yoctoproject.org/docs/2.6/ref-manual/ref-manual.html#var-DISTRO[distro]. If you are using "poky", the default distro in Yocto, you can change it in your conf/local.conf to "poky-sota". Alternatively, if you are using your own or a third-party distro configuration, you can add `INHERIT += " sota"` to it, thus combining the capabilities of your distro with meta-updater features.
+3.  Set up your https://www.yoctoproject.org/docs/2.6/ref-manual/ref-manual.html#var-DISTRO[distro]. If you are using "poky", the default distro in Yocto, you can change it in your conf/local.conf to "poky-sota". Alternatively, if you are using your own or a third-party distro configuration, you can add the following parameters to it, thus combining the capabilities of your distro with meta-updater features.
```

Comment:
We should perhaps suggest `poky-sota-systemd` in addition to `poky-sota`.

Otherwise, looks fine to me!

Reply:
@patrickvacek do you also need to add the parameter `st poky-sota-systemd` in addition to `poky-sota` in the local.conf file or is `poky-sota-systemd` an alternative of changing it to `poky-sota`?

Reply:
Sorry! It's an alternative. You can only have one `DISTRO` so in this case we should suggest `poky-sota` or `poky-sota-systemd`.

Reply:
thank you for the clarification. I'll add it.